LayoutTests/fast/scrolling/mac/smooth-scroll-recursive-frame-to-frame.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true EventHandlerDrivenSmoothKeyboardScrollingEnabled=true ] -->
+
+<html>
+
+<head>
+    <script src="../../../resources/ui-helper.js"></script>
+    <script src="../../../resources/js-test-pre.js"></script>
+    <meta name="viewport" content="initial-scale=1.5, user-scalable=no">
+    <style>
+        body {
+            height: 2000px;
+        }
+        #iframe {
+            width: 20%;
+            height: 20%;
+            border: 1px solid black;
+            padding: 10px;
+        }
+    </style>
+    <script>
+        if (window.testRunner) {
+            testRunner.dumpAsText();
+            testRunner.waitUntilDone();
+        }
+
+        function documentScrolled()
+        {
+            debug("SUCCESSFUL: frame to frame recursive scroll.");
+            testRunner.notifyDone();
+        }
+
+        async function runTest()
+        {
+            if (!window.testRunner || !testRunner.runUIScript)
+                return;
+
+            var iframe = document.getElementById("iframe");
+            document.addEventListener("scroll", documentScrolled);
+
+            iframe.contentWindow.scrollTo(0,732);
```

Comment:
Is 732 what the height currently happens to be? If so, that's sensitive to future layout changes. I suggest just some big number that is much larger than the content size, or make the contents be something with specified height using CSS like `body { height: 2000px; }`